src/@types/i18next.d.ts

```diff
@@ -0,0 +1,17 @@
+// import the original type declarations
```

Comment:
Does this implementation work out of the box? Please double check if it is required to add `src/@types` path to the `tsconfig.json`.

Reply:
added `include src` to `tsconfig.json`